shared/oae/api/oae.core.js

```diff
@@ -84,7 +85,9 @@ define([
         'markdown',
         'sockjs',
         'tinycon',
-        'underscore'
+        'underscore',
+        'moment',
+        'cookies'
```

Comment:
Do we also need select2&async here?

Reply:
I honestly don't know. It works without it, loading those selectively when needed 🤷‍♂️